verification/certifyKey_test.go

```diff
@@ -14,19 +15,43 @@ import (
 	"github.com/zmap/zlint/v3/lint"
 )
 
-// This file is used to test the certify key command by using a simulator
+// This file is used to test the certify key command by using a simulator/emulator
+
+func GetTestTarget_CertifyKey() ([]TestDPEInstance, error) {
```

Comment:
If you make all the below changes, I think this function should be reusable by all tests (instead of being specific to each command)

Reply:
GetTestTarget methods from each test file will invoke the GetEmulatorTarget which is a common function in verification_test.go and it is reused by all the command tests.

Reply:
Addressed as per your comment